pkg/graph/schema.graphql

```diff
@@ -1398,6 +1408,9 @@ unlockAllTaskListSections(modelPlanID: UUID!): [TaskListSectionLockStatus!]!
 
 updatePlanPayments(id: UUID!, changes: PlanPaymentsChanges!): PlanPayments!
 @hasRole(role: MINT_USER)
+
+acceptNDA(accept: Boolean!): NDAInfo!
```

Comment:
Should we also catch instances of `accept` and convert them to `agree` here?

Reply:
hmmm I'm not sure for this one here? agreeNDA, doesn't sound right to me, and if we call it acceptNDA, I think the parameter should say accept as well. Thoughts? I think the other sections do make sense to say agree

Reply:
Or `agreeToNDA`? Then `agree: true` sorta makes sense?

Maybe the bigger question lies un-asked: Do we need `accept: Boolean!` at all? I would assume if you're calling this mutation you're always accepting it, perhaps we can free two birds with one key?

Reply:
Hmm, that is a good question, I was approaching it so that you could later remove your agreement by passing false. BUT to your point, would we ever give the end user the option to revoke their agreement? ( I don't think so from the FIGMA).

Reply:
Same. Maybe we just hard-code the value to true and simplify the resolver logic to match? We can always add this later if it is a need, but right now means that we'd probably want to test all the logic/behvaior for passing `false` when we might not have a need for that.

Reply:
True, I actually already created Unit tests for passing false, so I have the the test cases, covered.  

I think it might just make sense to leave the agree param, but set a default GQL parameter value to true.
`agreeToNDA(agree: Boolean! = true): NDAInfo!`

It would still simplify the action for the frontend, but leave the logic in the backend in case it's useful later (since the test cases etc are already built out).

Reply:
I made the change with the default value, does this approach work for you @ClayBenson94 ?
```gql
mutation AgreeToNDA 
{
    agreeToNDA{
      agreed
      agreedDts
    }
}
```